streams/iq.m3u

```diff
@@ -153,8 +153,6 @@ https://livefa.marjaeyattv.com/mtv_ar/playlist.m3u8
 https://livefa.marjaeyattv.com/mtv_en/playlist.m3u8
 #EXTINF:-1 tvg-id="MarjaeyatTVPersian.iq",Marjaeyat TV Persian (240p) [Not 24/7]
 https://livefa.marjaeyattv.com/mtv_fa/playlist.m3u8
-#EXTINF:-1 tvg-id="",Nabaa TV
-https://5dc7d824154d0.streamlock.net/live/Nabaa/playlist.m3u8
```

Comment:
Move to https://github.com/iptv-org/iptv/pull/14353/files

Reply:
Confirmed. https://www.lyngsat.com/tvchannels/lb/Nabaa-TV.html

Most certainly was being mistaken with "Al Nabaa TV", which is indeed an Iraqian channel, but the website and its server have been seized by the FBI and the american cybersec.